util/ipgen/renderer.py

```diff
@@ -103,7 +103,7 @@ def get_uniquified_name(self, name: str) -> str:
         return self.ip_config.param_values["uniquified_modules"].get(name)
 
     def _replace_uniquified_prefix(self, core_name: str) -> str:
-        uniquified_modules = self.ip_config.param_values["uniquified_modules"]
+        uniquified_modules = self.ip_config.param_values.get("uniquified_modules", {})
```

Comment:
I wonder if we should make this something that doesn't require a template parameter. That would be for another PR, though.